n2survey/lime/survey.py

```diff
@@ -229,6 +237,32 @@ def read_responses(self, responses_file: str) -> None:
         self.responses = question_responses
         self.lime_system_info = system_info
 
+    def __copy__(self):
+        """Create a shallow copy of the LimeSurvey instance
+
+        Returns:
+            LimeSurvey: a shallow copy of the LimeSurvey instance
+        """
+        survey_copy = LimeSurvey()
+        survey_copy.read_structure(self.structure_file)
```

Comment:
Would it work if we just remove this line? As far as I understand, it would. That would be better since otherwise we did not make any difference from the previous version